types/imapflow/index.d.ts

```diff
@@ -293,7 +296,7 @@ export interface SearchObject {
 
 export interface StatusObject {
```

Comment:
You can extend this interface from StatusQuery. The interface has the same properties.

Reply:
```ts
export interface StatusObject {
    path: string;
    messages?: number;
    recent?: number;
    uid?: number;
    uidValidity?: bigint;
    unseen?: number;
    highestModSeq?: bigint;
}


export interface StatusQuery {
    messages?: boolean;
    recent?: boolean;
    uidNext?: boolean;
    uidValidity?: boolean;
    unseen?: boolean;
    highestModseq?: boolean;
}
```
They have a different structure.

```ts
export interface StatusObject {
  path: string;
  uid?: number;
}

export interface StatusQuery {
  uidNext?: boolean;
}
```

Therefore, they cannot be  StatusQuery extends StatusObject or StatusObject extends StatusQuery.